templates/barcode-development.ini.erb

```diff
@@ -0,0 +1,6 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+barcode_key = "<%= @barcode_key %>"
```

Comment:
actually i just noticed that myself.  the key is, the other two aren't and we should kill the quotes